/*
Nested Try Catch block:-

* We can also use try catch block inside catch block. but a try must be followed by atleast one 
catch block.

* try catch block is also possible inside try block.
* try catch block is necessary whenever exception is possible.

*/



class NestedTryCatch
{
	
	
}